front/src/modules/trainschedule/components/DriverTrainSchedule/DriverTrainSchedule.tsx

```diff
@@ -1,76 +0,0 @@
-import { useMemo } from 'react';
-
-import type {
-  PathPropertiesFormatted,
-  SimulationResponseSuccess,
-} from 'applications/operationalStudies/types';
-import type { LightRollingStock, TrainScheduleBase } from 'common/api/osrdEditoastApi';
-import { Loader } from 'common/Loaders';
-
-import { BaseOrEco, type BaseOrEcoType } from './consts';
-import DriverTrainScheduleHeader from './DriverTrainScheduleHeader';
-import DriverTrainScheduleStopList from './DriverTrainScheduleStopList';
-import type { OperationalPointWithTimeAndSpeed } from './types';
```

Comment:
This type should be moved in `operationalStudies/types.ts`

Reply:
Done

Reply:
Done